pallets/detach/src/lib.rs

```diff
@@ -174,10 +187,14 @@ pub struct Proto<TAccountId, TBlockNumber> {
 	pub metadata: BTreeMap<Vec<u8>, Hash256>,
 }
 
+/// Struct that represents a Proto-Fragment that was detached (i.e exported) onto another Blockchain (e.g Ethereum)
 #[derive(Encode, Decode, Clone, scale_info::TypeInfo, Debug, PartialEq)]
 pub struct ExportData {
+	/// The Blockchain the Proto-Fragment was detached onto
 	chain: SupportedChains,
+	/// The account address (on the blockchain `SupportedChain`) that the Proto-Fragment was transfered into
 	owner: Vec<u8>,
+	/// ¿
```

Comment:
For now we don't allow to re-attach but in the future we will, this nonce is in 1:1 relationship with the remote chain, so that e.g. if we detach on ethereum the message cannot be repeated and needs to go 1:1 with clamor